lib/shoryuken/queue.rb

```diff
@@ -52,6 +49,21 @@ def fifo?
 
     private
 
+    def set_name_and_url(name_or_url)
+      if name_or_url.start_with?('https://sqs.')
```

Comment:
@dolan just tested myself, and the AWS SDK does not support `sqs:` protocol, this a protocol made up by Convox. 

> as long as everyone knows to do that.

as long as Convox users know that the Convox makes up protocols and that's invalid in the AWS SDK context.